portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFields.testcase

```diff
@@ -663,4 +663,73 @@ definition {
 			fieldValidationErrorMessage = "This field is required.");
 	}
 
+	@description = "LPS-182981 - Verify if when adding a new option in a field this option is added only in the chosen field "
+	@priority = 4
+	test ViewOptionAddedOnlyInTheChosenField {
+		property portal.acceptance = "true";
```

Comment:
@alanafergs Let's be mindful of what we want to add to acceptance. In theory, we want to make sure this test is stable on CI before we add it to acceptance to avoid having flaky tests enter acceptance.